code/datums/hostility/hostility.dm

```diff
@@ -0,0 +1,72 @@
+// A small object that is useful for recycling hostility logic in things such as mobs or turrets.
+
+/datum/hostility
+	var/datum/holder = null
```

Comment:
Done. Left securitron hostility without reference to a holder's variables because it's currently unused and I don't feel like refactoring secbot.dm at the moment.